### PR TITLE
simplify parameter to ConnectionConfiguration

### PR DESCRIPTION
**Pull request by adamralph**

### CODE REVIEW

File: src/NServiceBus.RabbitMQ.Tests/RabbitMqContext.cs [hunk 1]
```diff
@@ -49,21 +48,20 @@ public void SetUp()
             routingTopology = new ConventionalRoutingTopology(true);
             receivedMessages = new BlockingCollection<IncomingMessage>();
 
-            var settings = new SettingsHolder();
-            settings.Set("NServiceBus.Routing.EndpointName", "endpoint");
+            var endpointName = "NServiceBus.Routing.EndpointName";
```
Comment (bording) [inline src/NServiceBus.RabbitMQ.Tests/RabbitMqContext.cs]: To match previous behavior, this should be set to "endpoint" instead.

However, I think an even better option would be to remove `endpointName` completely and pass `ReceiverQueue` directly into the parser instead. That actually matches the real behavior better anyway.

File: src/NServiceBus.RabbitMQ.Tests/ConnectionString/ConnectionStringParserTests.cs [hunk 2]
```diff
@@ -12,19 +11,18 @@ public class ConnectionStringParserTests
...
 
-        SettingsHolder settings;
+        string endpointName;
 
         [SetUp]
         public void Setup()
         {
-            settings = new SettingsHolder();
-            settings.Set("NServiceBus.Routing.EndpointName", "endpoint");
+            endpointName= "NServiceBus.Routing.EndpointName";
```
Comment (bording) [inline src/NServiceBus.RabbitMQ.Tests/ConnectionString/ConnectionStringParserTests.cs]: This should be set to "endpoint" to match the previous behavior.

Instead, `endpointName` can become a const string, and we can drop the setup method entirely.

File: src/NServiceBus.RabbitMQ.Tests/ConnectionString/ConnectionConfigurationTests.cs [hunk 3]
```diff
@@ -13,10 +12,7 @@ public class ConnectionConfigurationTests
         [SetUp]
         public void Setup()
         {
-            var settings = new SettingsHolder();
-            settings.Set("NServiceBus.Routing.EndpointName", "endpoint");
-
-            defaults = new ConnectionConfiguration(settings);
+            defaults = new ConnectionConfiguration("NServiceBus.Routing.EndpointName");
```
Comment (bording) [inline src/NServiceBus.RabbitMQ.Tests/ConnectionString/ConnectionConfigurationTests.cs]: This should be using "endpoint" to match the previous behavior.

File: src/NServiceBus.RabbitMQ.Tests/RabbitMqContext.cs (hunk 1 above)
Comment (adamralph) [inline src/NServiceBus.RabbitMQ.Tests/RabbitMqContext.cs]: Oops, I used the wrong name! Will fix.

I'll make the `ReceiverQueue` change in a separate commit.

File: src/NServiceBus.RabbitMQ.Tests/ConnectionString/ConnectionStringParserTests.cs (hunk 2 above)
Comment (adamralph) [inline src/NServiceBus.RabbitMQ.Tests/ConnectionString/ConnectionStringParserTests.cs]: 👍 will fix the string and send the const change in a separate commit.

**Comment by adamralph** [comment]
@bording correct string fixup! and further simplification pushed.

**Comment by bording** [comment]
LGTM

Mind rebasing against latest develop before we merge?

**Comment by adamralph** [comment]
Sure, will do.

**Comment by adamralph** [comment]
Rebased.

**Comment by adamralph** [comment]
Rebased again. (And remembered to squash this time!)
